rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_11_13.md

```diff
@@ -0,0 +1,55 @@
+# Proposal to upgrade the ckBTC minter canister and reduce minimum retrieval amount
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `bcbccf79c89a3e81b1a38d8233f8f81d1af1b245`
+
+New compressed Wasm hash: `e90d3f24a6eb3e1da8f42f3160756c40598e2a6783097db1beeb31873224e787`
+
+Upgrade args hash: `0b7d4ab8fe7adf7c609c7d565765abfa309ba7d03660439ec12c2e08e33ee6a5`
+
+Target canister: `mqygn-kiaaa-aaaar-qaadq-cai`
+
+Previous ckBTC minter proposal: https://dashboard.internetcomputer.org/proposal/133840
+
+---
+
+## Motivation
+Reduce the minimum ckbtc retrieval amount to 0.0005 BTC, as proposed by a recently passed motion proposal https://dashboard.internetcomputer.org/proposal/133462.
+
+The previous proposal https://dashboard.internetcomputer.org/proposal/133840 failed to change this setting because the ckBTC minter implementation was using a hardcoded value.
+A fix has since been implemented, and this upgrade will apply the fix.
+
+## Upgrade args
+
+```
+git fetch
+git checkout bcbccf79c89a3e81b1a38d8233f8f81d1af1b245
+cd rs/bitcoin/ckbtc/minter
+didc encode -d ckbtc_minter.did -t '(MinterArg)' '(variant {Upgrade = opt record { retrieve_btc_min_amount = opt 50000 }})' | xxd -r -p | sha256sum
```

Comment:
Should be (same as [previous proposal](https://dashboard.internetcomputer.org/proposal/133840))
`0b7d4ab8fe7adf7c609c7d565765abfa309ba7d03660439ec12c2e08e33ee6a5`

Reply:
I got the same.